starknet_py/tests/e2e/docs/guide/test_using_existing_contracts.py

```diff
@@ -86,3 +88,26 @@ async def test_using_existing_contracts(account, erc20_contract):
     # docs: end
 
     assert balance == 200
+
+    # docs-raw-call: start
+    from starkware.starknet.public.abi import get_selector_from_name
+
+    from starknet_py.net.client_models import Call
+
+    eth_token_address = (
+        0x049D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
+    )
+    # docs-raw-call: end
+    eth_token_address = int(FEE_CONTRACT_ADDRESS, 0)
+    # docs-raw-call: start
```

Comment:
Why do we have to do this? Isn't the first line enough?

Reply:
We don't, I thought it could be a little change-proof this way, as the `FEE_CONTRACT_ADDRESS` constant could change, and I wanted to show some address in the docs. Should we somehow change it?

Reply:
I would leave that `0x049D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7` address only. If it changes we will know about it.